schema/ecu_schema.md

```diff
@@ -0,0 +1,10 @@
+Define message ID range - this is checked against TX message IDs:
+
+```json
+{
+    "message_id_min": 100,
+    "message_id_max": 200
+}
+```

```

Comment:
Is this the message ID range for a given ECU? Or is it the range for all ECUs on a single network?

Reply:
good question - it's for a given ECU, i'll clarify

- [ ] clarify that range is for ECU

Reply:
talked to prom - we're getting rid of board-level MID ranges and moving towards message-level MID priorities

the default priority for a message will be the lowest priority. you can specify a higher priority for a message

the user will be able to specify the priority range they want - 3 levels, 5 levels, etc.

the application will have some deterministic way of assigning message IDs in a packed order. there needs to be a minimum message ID to autogen so that we don't autogen all over the inverter messages. we will also allow the user to override message ID, but then they can't use priority

feel free to comment @gtaharaedmonds

Reply:
interesting - are you suggesting that instead of specifying an ID for each message, we just specifiy a priority and the autogen will figure out the rest?